handlers/libhoney_event_handler.go

```diff
@@ -155,7 +158,10 @@ func (handler *libhoneyEventHandler) handleEvent(event assemblers.HttpEvent) {
 		ev.AddField(string(semconv.HTTPRequestBodySizeKey), event.Request.ContentLength)
 		if handler.config.IncludeRequestURL {
 			requestURI = event.Request.RequestURI
-			ev.AddField(string(semconv.URLPathKey), requestURI)
+			urlShape, err := handler.urlParser.Parse(requestURI)
```

Comment:
How do you feel about always adding the templated path and then using the new `IncludeRequesURL` config option to add in the original url?

Reply:
Templated path is not something we've got yet without accepting regexes from the user to know how to parse out the variable bits of a path. 😞